src/pybind/Makefile.am

```diff
@@ -7,8 +7,8 @@ if WITH_CYTHON
 
 PY_DISTUTILS = \
 	CPPFLAGS="-iquote \${abs_srcdir}/include ${AM_CPPFLAGS} ${CPPFLAGS}" \
-	CFLAGS="-iquote \${abs_srcdir}/include ${AM_CFLAGS} ${CFLAGS}" \
-	LDFLAGS="-L\${abs_builddir}/.libs $(subst -pie,,${AM_LDFLAGS}) ${LDFLAGS}" \
+	CFLAGS="-iquote \${abs_srcdir}/include ${AM_CFLAGS} ${PYTHON_CFLAGS}" \
+	LDFLAGS="-L\${abs_builddir}/.libs $(subst -pie,,${AM_LDFLAGS}) ${PYTHON_LDFLAGS}" \
```

Comment:
I think this makes sense, since we're compiling these to run as modules loaded by python, and if python has different cflags/ldflags than ceph it may create an incompatibility, like james saw